setup.py

```diff
@@ -15,7 +15,7 @@
 
 install_requires = [
     "ply>=3.4,<4.0",
-    "six>=1.15.0"
+    "six~=1.15.0",
```

Comment:
It should be `six~=1.15`, the `six~=1.15.0` have different meanings. See here: https://www.python.org/dev/peps/pep-0440/#examples